controller/program/src/utils.rs

```diff
@@ -0,0 +1,63 @@
+use inf1_ctl_jiminy::{
+    instructions::admin::{add_lst::AddLstIxAccs, remove_lst::RemoveLstIxAccs},
+    keys::SYS_PROG_ID,
+};
+use jiminy_cpi::{
+    account::Abr,
+    program_error::{ProgramError, INVALID_ACCOUNT_DATA},
+};
+use jiminy_entrypoint::account::AccountHandle;
+use jiminy_sysvar_rent::Rent;
+use sanctum_system_jiminy::sanctum_system_core::instructions::transfer::{
+    NewTransferIxAccsBuilder, TransferIxData,
+};
+
+use crate::Cpi;
+
+pub fn pay_for_rent_exempt_shortfall(
+    abr: &mut Abr,
+    cpi: &mut Cpi,
+    handles: AddLstIxAccs<AccountHandle>,
```

Comment:
this should be `TransferIxAccs` to better generalize to other instructions; its not always that we're paying rent for `lst_state_list`